tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -74,14 +74,18 @@ def module_repos_collection(module_org, module_lce):
 
 
 @fixture
-def vm_content_hosts(request, module_repos_collection):
+def vm_content_hosts(request, module_loc, module_repos_collection):
     clients = []
     for _ in range(2):
         client = VirtualMachine(distro=module_repos_collection.distro)
         clients.append(client)
         request.addfinalizer(client.destroy)
         client.create()
         module_repos_collection.setup_virtual_machine(client)
+        # update client host location
+        host = entities.Host().search(query={'search': 'name={0}'.format(client.hostname)})[0]
```

Comment:
any way to make that change global? I looked through setup methods, but do not see exact place to use location. We will need update host with location in future for sure, so if that is easy to do - that will be great

Reply:
@oshtaier Good idea , will investigate on that.